src/types/pathways-model.d.ts

```diff
@@ -60,6 +54,8 @@ declare module 'pathways-model' {
       cql: string;
       elm?: ElmLibrary;
     };
+    criteria?: string;
```

Comment:
So I know this isn't linked to the criteria provider yet but we probably want to set the criteria string to be the `transition.condition.cql` and then use that in the future task to populate the `transition.condition.elm`

Reply:
Some of it can be done in Pathways-261 but my main point was we don't need to create a new property in the interface for criteria when one already exists for that use.

Reply:
removed criteria property